app/cdap/components/CreateAndEditRecipeForm/styledComponents.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import styled from 'styled-components';
+import { TextField, Typography, Box, Button } from '@material-ui/core';
+import { TextareaAutosize } from '@material-ui/core';
+import { grey, blue } from '@material-ui/core/colors';
+
+export const FormFieldWrapper = styled(Box)`
+  width: calc(100% - 60px);
+  margin-right: 60px;
+  margin-bottom: 15px;
+  margin-top: 15px;
+`;
+
+export const Label = styled(Typography)`
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 150%;
+  letter-spacing: 0.15px;
+`;
+
+export const ErrorLabel = styled(Label)`
+  color: #e05243;
+`;
+
+export const NormalLabel = styled(Label)`
+  color: #5f6368;
```

Comment:
can we reduce this component by adding this color as default in the component at line 29
then we can use ErrorLabel as just another variant

plz check if this works

Reply:
removed NormalLabel component and added this color in default component